main/webapp/modules/core/MOD-INF/controller.js

```diff
@@ -266,7 +269,7 @@ function registerImporting() {
   IM.registerMimeType("text/plain", "text/line-based");
   IM.registerMimeType("text/csv", "text/line-based/*sv");
   IM.registerMimeType("text/x-csv", "text/line-based/*sv");
-  IM.registerMimeType("text/tab-separated-value", "text/line-based/*sv");
+  IM.registerMimeType("text/tab-separated-values", "text/line-based/*sv");
```

Comment:
IANA confirmed at https://www.iana.org/assignments/media-types/text/tab-separated-values

Reply:
It's probably useful to keep the singular version around too: if it was added, it might be in use somewhere. There is no harm in having both.